.github/workflows/release-wheel.yml

```diff
@@ -0,0 +1,63 @@
+name: release-wheel
+
+# Following will be changed in due course
+on: [push, pull_request]
+
+concurrency: 
+  # this ensures after each commit the old jobs are cancelled and the new ones
+  # run instead.
+  group: ${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+env:
+  VERSION_FILE_NAME: version.txt
+  WHEEL_ARTIFACT: fairlearn-wheels
+
+jobs:
+  build-wheel:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: "3.10"
+    - name: Update pip and setuptools
+      run: |
+        pip install --upgrade pip
+        pip install --upgrade setuptools wheel
+    - name: Build wheels
+      run: python ./scripts/build_wheels.py --version-filename $VERSION_FILE_NAME
+    - name: Save wheels to artifact
+      uses: actions/upload-artifact@v3
+      with:
+        name: env.WHEEL_ARTIFACT
+        path: dist/
+
+  test-wheel:
+    needs: build-wheel
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        python-version: ['3.8', '3.9', '3.10']
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }} on ${{ matrix.os }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Remove Fairlearn source directory
+      uses: JesseTG/rm@v1.0.3
+      with:
+        path: fairlearn
+    - name: Download wheel artifact
+      uses: actions/download-artifact@v3
+      with:
+        name: env.WHEEL_ARTIFACT
+    - name: Install Fairlearn from wheel
+      run: python scripts/install_fairlearn_from_wheel.py --wheel-dir=.
```

Comment:
can this file not be removed and have here just  simple command?

Reply:
I spent some time trying to do that. However, to build the name of the `.whl` I'd have to figure out the version we're about to release.